Resources/Server Info/Guidebook/Chef/pitzah.xml

```diff
@@ -35,7 +35,7 @@
 - Cheese wedge
 </Box>
 <Box Orientation="Vertical">
-- 2x raw meat
+- 3x raw meat
```

Comment:
Check the recipe for meat pizza in the actual meal_recipes file. I changed it to 2.

Reply:
Okay, it's actually still on 3, even though I changed that to 2 in my last PR that got merged, and somehow that didn't go through.

So if you could change the meat pizza to 2 meat, that'd be swell.

Reply:
Sure thing